example/eventsource-polyfill.js

```diff
@@ -282,6 +282,35 @@ process.umask = function() { return 0; };
 
 /***/ }),
 /* 2 */
```

Comment:
I don't know why this file is checked in. It's just the output from running `npm run polyfill`, right? I would think the normal way to do this would be to just say that you have to run the build before running the sample.

Reply:
Yeah, just `npm run polyfill`. Seems reasonable to remove it from the repo, it gave me some confusion as it was in the initial clone, but not up to date.

Reply:
Let's leave it for now, and consider taking it out and updating the readme instructions as a separate PR. I just never noticed it before.